misc/services/mobile-webui/mobile-webui-frontend-testing/tests/spec/picking/picking_catchWeight.spec.js

```diff
@@ -21,6 +21,7 @@ const createMasterdata = async () => {
                     allowPickingAnyHU: true,
                     pickWithNewLU: true,
                     allowNewTU: true,
+                    allowCompletingPartialPickingJob: true,
```

Comment:
why changing allowCompletingPartialPickingJob to true?
what about regression?

Reply:
the way I see it, the test is focused on picking with catch weight, so the setting allowCompletingPartialPickingJob is not relevant for the actual picking... but I see your point, the default behaviour for the setting is false, so then it should be false, but that way I have to change the tests to make sure I pick the full qty so I can complete the job